pkg/config/config.go

```diff
@@ -61,14 +70,19 @@ func FindRegalDirectory(path string) (*os.File, error) {
 			}
 		}
 
-		if searchPath == rio.PathSeparator+regalDirName {
+		if searchPath == volume+rio.PathSeparator+regalDirName {
 			// Stop traversing at the root path
 			return nil, fmt.Errorf("can't traverse past root directory %w", err)
 		}
 
 		// Move up one level in the directory tree
 		parts := strings.Split(dir, rio.PathSeparator)
 		parts = parts[:len(parts)-1]
+
+		if parts[0] == volume {
+			parts[0] = volume + rio.PathSeparator
```

Comment:
Is this needed? I thought that file path.Join would be smart enough to do that?

Reply:
It did not, so you'd get `C:Users` rather than `C:\Users`. Maybe there's a cleaner way to do this, but this one at least works :)

Reply:
Good to know 😞